### PR TITLE
ROX-20748: Create tenant configuration for (existing) RH ACS tenant

### PR DESCRIPTION
**Pull request by tommartensen**
The RH ACS tenant is already in use and we are onboarding our components. 

In preparation for creating releases from Konflux, we are working through the "Onboarding Red Hat Products to Konflux" document. As a first step, we'd like to manage our existing IntegrationTestScenario, in which we provide a custom policy configuration to the non-hermetic EC. 

If relevant, the custom configuration lives at https://github.com/stackrox/stackrox/tree/rhtap-allow-quay-registry/.tekton/enterprise-contract, and we use it to whitelist the Golang builder and Cent OS (no RHEL RPM installable) as base images and allow custom tasks in the pipeline to prefetch content for our Scanner and Collector components. 

We'll follow this PR up with configuration for ReleasePlan, once we have figured out the the ReleasePlanAdmission. 

# TODO: 
- [x] MR for the EC policy: https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/merge_requests/184
- [x] Update policy configuration to namespace/name reference

### CODE REVIEW

**Comment by tommartensen** [comment]
I am not a member of the 'redhat-appstudio' and can't request review from @scoheb @ernesgonzalez33, so this comment must suffice :/

File: cluster/stone-prd-rh01/tenants/rh-acs-tenant/integration_test_scenario.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      - name: url
+        value: 'https://github.com/redhat-appstudio/build-definitions'
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: pipelines/enterprise-contract-redhat-no-hermetic.yaml
+    resolver: git
+  params:
+  - name: POLICY_CONFIGURATION
+    value: github.com/stackrox/stackrox?ref=rhtap-allow-quay-registry//.tekton/enterprise-contract
```
Comment (ernesgonzalez33) [inline cluster/stone-prd-rh01/tenants/rh-acs-tenant/integration_test_scenario.yaml:22]: Usually what we are doing with the policies is create them in the same repo as the ReleasePlanAdmission and reference them here using the `namespace/name` format of referencing resources in Kubernetes. I'm not sure how this works so I will let @scoheb check it out as well

Comment (tommartensen) [reply cluster/stone-prd-rh01/tenants/rh-acs-tenant/integration_test_scenario.yaml:22]: I found an example in https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/blob/main/config/OP/EnterpriseContractPolicy/registry-rhtas.yaml?ref_type=heads and will adapt it. I assume the changes there need to be merged and provisioned before this PR can be merged?

Comment (tommartensen) [reply cluster/stone-prd-rh01/tenants/rh-acs-tenant/integration_test_scenario.yaml:22]: Proposal for the policy: https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/merge_requests/184 .

Comment (scoheb) [reply cluster/stone-prd-rh01/tenants/rh-acs-tenant/integration_test_scenario.yaml:22]: @tommartensen https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/merge_requests/184 looks good. But I cannot approve. 

@ernesgonzalez33 you are bang on!

Comment (tommartensen) [reply cluster/stone-prd-rh01/tenants/rh-acs-tenant/integration_test_scenario.yaml]: @ernesgonzalez33 @scoheb 

The MR was approved and merged. I updated the policy configuration to namespace/name format. 
Can you confirm that it exists and approve this PR? 
Thanks.

**Pull request merged**